block_manipulation_service/src/main/java/com/stnetix/ariaddna/blockmanipulation/BlockGenerate.java

```diff
@@ -0,0 +1,107 @@
+package com.stnetix.ariaddna.blockmanipulation;
+
+import com.stnetix.ariaddna.commonutils.datetime.DateTime;
+import com.stnetix.ariaddna.commonutils.xmlparser.XmlParser;
+import com.stnetix.ariaddna.commonutils.xmlparser.exception.XmlParserException;
+import com.stnetix.ariaddna.commonutils.xmlparser.handlers.BlockManipulationSettingHandler;
+import com.stnetix.ariaddna.localstoragemanager.LocalService.LocalService;
+import com.stnetix.ariaddna.vufs.BusinessObjects.Block;
+import com.stnetix.ariaddna.vufs.BusinessObjects.Metafile;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.io.*;
+import java.net.URL;
+import java.nio.ByteBuffer;
+import java.nio.channels.FileChannel;
+import java.util.Date;
+import java.util.HashMap;
+
+/**
+ * Created by LugovoyAV on 15.02.2018.
+ */
+@Service
+public class BlockGenerate {
+    private final int defaultBlockSize = 5242880;
+    private int blockSize;
+
+    private HashMap<File,Long> files = new HashMap<>();
+
+    public int getBlockSize() {
+        return blockSize;
+    }
+
+    public BlockGenerate() {
+        if (!loadSettings())
+            blockSize = defaultBlockSize;
+    }
+
+    @Autowired
+    private LocalService localService;
+
+    public boolean loadSettings()
```

Comment:
So, I think that we don't use implementation load configuration per service. Because it make more code duplicates. We should use application configuration for load configuration all service in boot application. And other service (e.g. bms) can get from application configuration some settings late.

Reply:
create issue https://github.com/StnetixDevTeam/ariADDna/issues/130